mac/deploy_mac.sh

```diff
@@ -73,7 +73,7 @@ build_installer_image()
     local dmgbuild_bin="$(python -c 'import site; print(site.USER_BASE)')/bin/dmgbuild"
 
     # Get Jamulus version
-    local app_version="$(cat "${project_path}" | sed -nE 's/^VERSION *= *(.*)$/\1/p')"
+    local app_version="$(grep -oP 'VERSION = \K\w[^\s\\]*' Jamulus.pro)"
```

Comment:
This change from `sed -nE` to `grep -oP` breaks on Mac. See https://github.com/jamulussoftware/jamulus/runs/5247842643?check_suite_focus=true#step:6:1652

I've filed #2421 to fix this.